src/main/kotlin/uk/gov/justice/digital/hmpps/externalmovementsapi/sync/SyncTapApplication.kt

```diff
@@ -72,6 +59,5 @@ class SyncTapApplication(
     approvedAt = approvedAt,
     approvedBy = approvedBy,
     legacyId = movementApplicationId,
-    contact = contactPersonName,
```

Comment:
I've made a note in MIA-877 that Syscon will now send contactPersonName with their ScheduledTemporaryAbsenceResponse so we can map it to our occurrence